src/components/form.js

```diff
@@ -20,13 +20,13 @@ const Form = forwardRef(({ children, success, error, ...props }, ref) => {
     <StyledForm ref={ref} {...props}>
       {success && (
         <Notification title="Success" color="#B4FF39">
-          <p>{success}</p>
+          {success.length > 0 && <p>{success}</p>}
         </Notification>
       )}
 
       {error && (
-        <Notification title="Error" color={theme.colors.bavarian.red200}>
-          <p>{error}</p>
+        <Notification title="Error" color={theme.colors.error}>
+          {error.length > 0 && <p>{error}</p>}
```

Comment:
This is a bug I found from last year and sometimes an error isn't passed but it'll still render the p tag which creates unwanted spacing.

Reply:
I don't follow how an error would not be passed but the `p` would render -- the whole `Notification` is wrapped in a check that `error` is truthy? If you are confident we need this additional length check specifically, please add it to that existing conditional around this, otherwise we risk rendering the `Notification` with nothing in it?